src/app/config/projectProperties.json

```
@@ -150,5 +150,13 @@
         "word": {
             "displayname": "Word"
         }
+    },
+    "manifestTypes": {
+        "xml": {
+            "displayname": "XML Manifest"
+        },
+        "json": {
+            "displayname": "Unified manifest for Microsoft 365"
```

Marketing wants the "u" to be lower case so this is more a description than a name.

```suggestion
            "displayname": "unified manifest for Microsoft 365"
```